src/Consolonia.PlatformSupport/CursesConsole.cs

```diff
@@ -608,107 +612,70 @@ private void HandleMouseInput(Curses.MouseEvent ev)
                     case Curses.Event.ButtonShift:
                         // we ignore these, as they are picked up by input modifiers
                         break;
-                    case Curses.Event.Button1Clicked:
-                        RaiseMouseClickedEvent(RawPointerEventType.LeftButtonDown, RawPointerEventType.LeftButtonUp,
-                            1, point, rawInputModifiers);
-                        break;
-                    case Curses.Event.Button1DoubleClicked:
-                        RaiseMouseClickedEvent(RawPointerEventType.LeftButtonDown, RawPointerEventType.LeftButtonUp,
-                            2, point, rawInputModifiers);
-                        break;
-                    case Curses.Event.Button1TripleClicked:
-                        RaiseMouseClickedEvent(RawPointerEventType.LeftButtonDown, RawPointerEventType.LeftButtonUp,
-                            3, point, rawInputModifiers);
-                        break;
-                    case Curses.Event.Button2Clicked:
-                        RaiseMouseClickedEvent(RawPointerEventType.MiddleButtonDown,
-                            RawPointerEventType.MiddleButtonUp, 1, point, rawInputModifiers);
-                        break;
-                    case Curses.Event.Button2DoubleClicked:
-                        RaiseMouseClickedEvent(RawPointerEventType.MiddleButtonDown,
-                            RawPointerEventType.MiddleButtonUp, 2, point, rawInputModifiers);
-                        break;
-                    case Curses.Event.Button2TripleClicked:
-                        RaiseMouseClickedEvent(RawPointerEventType.MiddleButtonDown,
-                            RawPointerEventType.MiddleButtonUp, 3, point, rawInputModifiers);
```

Comment:
Should we exclude this from mousemask during initialization?

Reply:
yes, I updated the mousemask, good call